src/test/java/com/Futurecollars/Lekcja7/Zadanie7/TrainingTest.java

```diff
@@ -0,0 +1,30 @@
+package com.Futurecollars.Lekcja7.Zadanie7;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class TrainingTest {
+    @ParameterizedTest
+    @CsvSource({
+            "20, 250, 150, 1, 1, 3, 2, Bardzo dobra",
+            "45, 350, 165, 2, 2, 2, 1.7, Dobra",
+            "70, 500, 180, 3, 3, 1, 2.3, Bardzo dobra",
+            "40, 280, 170, 2, 1, 2, 1.7, Dobra"   })
+    void shouldtEvaluateTraining(int duration, int caloriesBurned, int AverageHeartRate, int gradeDuration,
+                                 int gradeCalories, int gradeAverageHeartRate, double AverageScale, String effectiveness)
+    {
+        Training training = new Training(duration,caloriesBurned,AverageHeartRate );
```

Comment:
zobacz jak wygląda twój kod, musisz po kilka razy przekazywać ten sam parametr do tego samego obiektu żeby wszystko się wyliczyło. Konieczny jest tutaj refactor. Ogólnie linie kodu odpowiedzialne za wyliczenie efektu treningu powinny się sprowadzać do: 
Training training = new Training(duration,caloriesBurned,AverageHeartRate );
String effectiveness = training.calculateEffectiveness(); 

gdzie w calculateEffectiveness() dzieie się wszystko to co u ciebie od lini 18-22 (uwzględniając jeszcze uwagi przekazane w innych komentarzach )